include/deal.II/base/trilinos_utilities.h

```diff
@@ -180,6 +184,32 @@ namespace Utilities
     duplicate_map(const Epetra_BlockMap &map, const Epetra_Comm &comm);
   } // namespace Trilinos
 #endif
+
+
+#ifdef DEAL_II_TRILINOS_WITH_TPETRA
+  namespace Trilinos
+  {
+    /**
+     * Creates and returns a
+     * <a
+     * href="https://docs.trilinos.org/dev/packages/teuchos/doc/html/classTeuchos_1_1RCP.html">Teuchos::RCP</a>
+     * object for type T.
+     *
+     * @note In Trilinos 14.0.0, the function
+     * <a
+     * href="https://docs.trilinos.org/dev/packages/teuchos/doc/html/namespaceTeuchos.html#a280c0ab8c9ee8d0481114d4edf5a3393">Teuchos::make_rcp()</a>
+     * was introduced.
+     */
+#  if defined DOXYGEN || !DEAL_II_TRILINOS_VERSION_GTE(14, 0, 0)
```

Comment:
I'm unsure whether this is necessary, but we seem to do it as follows everywhere else:
```suggestion
#  if defined(DOXYGEN) || !DEAL_II_TRILINOS_VERSION_GTE(14, 0, 0)
```

Reply:
Both should work. But I will adapt it, so it matches the rest.